sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample01_ConsumeEventsWithEventProcessor.cs

```diff
@@ -0,0 +1,229 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
I'd revert this;  I rewrote the first few samples already.

Reply:
I'm using it to double check I'm not breaking anything. We are not expected to have any major conflicts with your PR since the filenames are different, so we can simply delete it once the merge happens.